superset-frontend/src/utils/colorScheme.ts

```diff
@@ -31,6 +31,10 @@ import {
  */
 export const getColorNamespace = (namespace?: string) => namespace || undefined;
 
+export const ensureSharedLabelsColorsArray = (
```

Comment:
Probably because of the other "ensure" methods that transform  to a given type (Array for example), but this method seems to instead being doing a different thing with dicts by returning [], so we might want to consider a different name for it? or return a list with the dict values instead?

Reply:
Yeah, just that if I pass a dict {foo: "hello", bar: "world"} to something called ensure..Array I was expecting an array with the values and not an empty array? i.e, ["hello", "world"] instead of []. But probably a nit.

Reply:
Ok, I see your point. I'll keep it like this for lack of a better name at the moment.

Reply:
Maybe enforce?